include/cutlass/gemm/device/gemm_universal_adapter.h

```diff
@@ -60,6 +60,7 @@
 
 #if defined(CUTLASS_ENABLE_SYCL)
 #include "cutlass/util/sycl_event_manager.hpp"
+#include <syclcompat/syclcompat.hpp>
```

Comment:
Why is this needed? The include for syclcompat is already added via `cutlass/cutlass.h`

Reply:
Oh yes, that's true.
Removed in [7592346](https://github.com/codeplaysoftware/cutlass-fork/pull/106/commits/7592346d9510d7d23d4262c32ca0aba9322336c4)